twenty-first/src/shared_math/bfield_codec.rs

```diff
@@ -1524,9 +1525,56 @@ mod tests {
             }
         }
 
+        #[test]
+        fn encode_enum_with_uniform_data_size() {
```

Comment:
Would it make sense to turn the two new tests into `#[proptest]`s?

Reply:
Kind of. But I think we only need to run one test case here. More seems superfluous to me.